nsdb-http/src/test/scala/io/radicalbit/nsdb/web/QueryValidationApiTest.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2018 Radicalbit S.r.l.
```

Comment:
We should update the copyright at some point

Reply:
Good point. We are reaching the end of 2019. Maybe a dedicated PR to change all the headers is needed sooner than we think